.ci/pytorch/test.sh

```diff
@@ -278,6 +278,46 @@ else
   DYNAMO_BENCHMARK_FLAGS+=(--device cuda)
 fi
 
+test_perf_for_dashboard() {
+  TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  mkdir -p "$TEST_REPORTS_DIR"
+
+  local suite="$1"
+  shift
+
+  for dtype in amp float32; do
+    # Run accuracy test
+    # All the accuracy tests can be skipped once the CI accuracy checking is stable enough
+    for backend in eager aot_eager; do
+      python "benchmarks/dynamo/$suite.py" \
```

Comment:
Just curious, where are the performance tests for `eager` and `aot_eager`?  I see only performance tests for `inductor` below.  They are not needed?